cost/azure/reserved_instances/expiration/azure_reserved_instance_expiration.pt

```diff
@@ -95,17 +95,17 @@ script "js_filtered_reservations", type: "javascript" do
   code <<-EOS
     var content=[];
     var now = new Date();
+    var now_ms = now.getTime();
     var one_day=1000*60*60*24;
     var date1_ms = now.getTime();
 
-    for(var i=0; i < reservations.length ; i++){
+    for (var i = 0; i < reservations.length ; i++) {
       reservation = reservations[i]
-      var end_date = (new Date(reservation['expiration_date'])).toISOString().slice(0, 10);
       var date2_ms = (new Date(reservation['expiration_date'])).getTime();
       var difference_ms = date2_ms - date1_ms;
       var daysLeft = Math.round(difference_ms/one_day);
 
-      if(daysLeft <= param_days_expiration) {
+      if (daysLeft <= param_days_expiration && now_ms <= date2_ms) {
```

Comment:
Hi @midir99 - I am assuming this has been tested in a sandbox tenant? Only reason I am saying is because sometimes the policy engine can be a bit funny when using the DateTime functions

fyi - @taniaSol

Reply:
Hey @nia-vf1 !

Yep, this was tested in a sandbox tenant, I made sure it worked! :)